### PR TITLE
Displayed ship info refinements from @stechio.

### PR DESCRIPTION
**Pull request by quyykk**
**Feature:** This PR is a continuation from #4484 from @stechio.

## Feature Details
This PR improves the attribute list for ships by changing how they are displayed. See the screenshots for comparisons. All credit goes to @stechio.

## UI Screenshots

Left is continuous, right is this PR.

pelican-shipyard-master pelican-shipyard-stechio

### CODE REVIEW

**Comment by Zitchas** [comment]
All in all, this looks good.

The one detail that I'm not fond of is the change from "Required Crew/Bunks" to "Crew (min-max)".

I know many people are capture focused, but to many of us (and, arguably, to the lore) bunks aren't "places to put crew that might happen to be empty so you can put a passenger there", but rather "places to put passengers that you can put a soldier if you want."

The difference is slight, but given that there is no mechanical benefit to having additional crew (aside from capturing), I think it is more accurate to say that bunks are things that hold people that you just happen to be able to fill with crew.

Basically, I prefer the look of having it list "Here's how many required crew" and "here's the max bunks". It makes more sense to me.

**Review by tehhowch — CHANGES_REQUESTED** [review]
Please ensure the commit is properly attributed with a co-author trailer. One way to do that would be to squash the original work into a single commit and then commit the merge & additional feedback resolution

**Comment by quyykk** [comment]
Shoot I pushed the wrong branch. Anyways, I squashed their commits into one and then merged into master. Now it says for that commit "stechio authored and quyykk committed". Is that okay?

**Comment by Amazinite** [comment]
I'd just do what you had just before you squashed it: maintain the commit history of the old PR. That's what I did for #5971 when picking up #3502.

**Comment by quyykk** [comment]
There you go :)

**Comment by quyykk** [comment]
> The one detail that I'm not fond of is the change from "Required Crew/Bunks" to "Crew (min-max)".

I agree. How about `crew (required - total)`? `crew (required - free)`? `crew (required - max)`? I like the first one.

**Comment by Zitchas** [comment]
> > The one detail that I'm not fond of is the change from "Required Crew/Bunks" to "Crew (min-max)".
> 
> I agree. How about `crew (required - total)` or `crew (required - max)`?

I don't see either of those as really being changes. It's not so much the format that I'm concerned about, but rather the "Bunks" reference. We have outfits that talk about bunks, we have missions and jobs that talk about needing x bunks, we have `<bunks>` as a replacement text. I think as a matter of consistency and clarity to the player we should continue to refer to the maximum number of people onboard as bunks.

**Comment by quyykk** [comment]
Well, what would you suggest? To keep with the new formatting `bunks (required - total)`?

**Comment by Zitchas** [comment]
Just looking at the pictures, and it strikes me that the most appropriate format could well be that of the other space related things:

`bunks free    31/75`

After all, the number of required crew is irrelevant, right? What players are really concerned about is generally one of two values:

1. How many free bunks to put passengers in, or
2. What is the total number of bunks to attack an enemy ship with.

**Comment by quyykk** [comment]
It now looks like this, as proposed by Zitchas. The relevant change is the "bunks free" line. It takes some getting used to, but I think it's an improvement.

stechio bunks free

File: data/tooltips.txt [hunk 1]
```diff
@@ -329,8 +329,8 @@ tip "radar jamming:"
 tip "ramscoop:"
 	`Replenishes fuel by harvesting the stellar wind. Ramscoops are more effective when closer to a star. If you add more than one ramscoop, each additional one is less effective: you need four ramscoops to achieve double the recharge rate that one ramscoop provides.`
 
-tip "required crew / bunks:"
-	`How many crew members are required to operate this ship, and how many total bunks are available. Extra bunks can be used for passengers or extra crew.`
+tip "bunks free:"
```
Comment (Amazinite) [inline data/tooltips.txt:332]: Not too keen on this change. While it does flip the order to display the number of bunks free, which is more useful for running jobs than the number of required crew, this now doesn't communicate why some bunks are taken up. I think we should just leave this line how it was, displaying the number of required crew and the total number of bunks.

**Review by Amazinite — CHANGES_REQUESTED** [review]
